tests/samples.ts

```diff
@@ -4,7 +4,26 @@ import {
    ApplicationLoadBalancerEventRequestContext,
    APIGatewayRequestEvent,
    ApplicationLoadBalancerRequestEvent } from '../src/request-response-types';
-import { Context } from 'aws-lambda';
+import { APIGatewayEventIdentity, Context } from 'aws-lambda';
+import withDefault from './test-utils/withDefault';
+
+interface MakeAPIGatewayRequestEventContextParams {
+   httpMethod?: string | null;
```

Comment:
It doesn't look like the aws-lambda types allow null for `httpMethod`.
```suggestion
   httpMethod?: string;
```